offload_data/plugin.cpp

```diff
@@ -111,7 +111,7 @@ class offload_data : public plugin {
 		for (auto& container_it : _offload_data_container)
 		{
 			// Get collecting time for each frame
-			_time_seq.push_back(container_it->offload_time);
+			_time_seq.push_back(std::chrono::duration_cast<std::chrono::duration<long, std::milli>>(container_it->offload_duration).count());
```

Comment:
The `.count()` will return a `long` whereas `_time_seq` is of type `int`. Please change `_time_seq` to a `std::vector<long>`.